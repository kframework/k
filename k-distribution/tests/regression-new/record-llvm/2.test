incrementBar(test(1, 0))
