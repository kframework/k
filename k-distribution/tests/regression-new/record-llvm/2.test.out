<k>
  test ( 1 , 1 )
</k>
